server/routes/appointments/create-and-edit/handlers/reviewPrisonersAlerts.ts

```diff
@@ -50,6 +56,13 @@ export default class ReviewPrisonersAlertsRoutes {
       return res.redirectOrReturn('date-and-time')
     }
 
+    if (config.nonAssociationsAppointmentReviewEnabled) {
```

Comment:
I think you also need to check there are more than one prisoner as call to nonAssociations is irrelevant.

Is it worthwhile having this in `reviewNonAssociations.ts` to avoid calling the "between" endpoint twice and redirect accordingly?